timecard.js

```diff
@@ -272,6 +273,58 @@ TimeCard.prototype.print = function(cb) {
 
 };
 
+/**
+ * Append .timecard.js to .gitignore file
+ *
+ */
+TimeCard.prototype.append = function(cb) {
+
+	var self = this;
+
+	// First check to see if the .gitignore file exists
+    fs.stat(process.cwd()+'/.gitignore', function(err, stats){
+
+    	if (err && err.code === 'ENOENT'){
+    		if (typeof cb === 'function') {
+    			cb(new Error(errors.noGitIgnoreFound));
+            	return;
+    		}
+        } else if (err) {
+            throw err;
+        }
+
+        if (stats && stats.isFile()){
+
+            var question = [{
+                    type: "confirm",
+                    name: "appendToGitIgnore",
+                    message: "A .gitignore file exists. Do you want to add .timecard.json to it?",
```

Comment:
Do you think we should just ask `'Do you want your timecard to be kept in version control?'` and then if they answer `yes`, you can report `.timecard.json' was added to your .gitignore file`